App/test/app/TestBookingConfirmation.java

```diff
@@ -0,0 +1,67 @@
+package app;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.sql.Date;
+import java.text.SimpleDateFormat;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.io.File;
+
+import java.text.ParseException;
+
+import static org.junit.Assert.assertEquals;
+
+
+public class TestBookingConfirmation {
+
+    BookingConfirmation bookingConfirmation;
+
+    @Test
+    public void testBooking() {
+        System.out.println(bookingConfirmation.Description);
+        System.out.println(bookingConfirmation.rooms_to_book);
+        System.out.println(bookingConfirmation.days);
+        System.out.println(bookingConfirmation.tariff);
+        System.out.println(bookingConfirmation.number_of_rooms);
+        assertEquals(bookingConfirmation.hotel_name, "TestName");
+        assertEquals(bookingConfirmation.HID, 1);
+        assertEquals(bookingConfirmation.Address, "TestAddress");
+        assertEquals(bookingConfirmation.City, "TestCity");
+        assertEquals(bookingConfirmation.tariff, 1);
+        assertEquals(bookingConfirmation.days, 3);
+    }
```

Comment:
In this bit you didn't anything referring into. Which means bookingConfirmation is on a Null Value

Reply:
In addition, this is not the way you test. You only call the parameter in the bookingConfirmation.